tests/unit/services/test_package.py

```diff
@@ -25,8 +25,8 @@
 
 def test_pack(package_service, default_factory, mocker):
     mock_pack_snap = mocker.patch.object(pack, "pack_snap")
-    mock_linters_run = mocker.patch.object(linters, "run_linters")
-    mock_linters_report = mocker.patch.object(linters, "report")
+    _ = mocker.patch.object(linters, "run_linters")
+    _ = mocker.patch.object(linters, "report")
```

Comment:
Can it just be `mocker.patch.object(`? Or it will also have some lint errors

Reply:
You're right - fixed!